bindings/bindingtester/cbatrace/cbatracefdb_c.g.sym

```diff
@@ -0,0 +1,55 @@
+// ////////////////////////////////////////////////////////////////////////////////////// //
```

Comment:
Same comment as above: I would prefer this not to be checked in (though for a code-review it's actually super useful!)